examples/graphql-api/app/styles/index.css

```diff
@@ -0,0 +1,74 @@
+a,
```

Comment:
Do you need this, and other inline, styles? So far examples are really simple and to the point, I think just using a `ul` and `li` for the list of characters with the browser default styles would be enough to show the point of the example

Reply:
Sure we can trim all that down here and the other bits as well.